src/PhaseState.jl

```diff
@@ -190,4 +190,26 @@ export getkfkrev
     end
     return kfs,krev
 end
+@inline function getkfkrevs(phase::U,T::W1,P::W2,C::W3,N::W4,ns::Q1,Gs::Array{Q2,1},diffs::Q3,V::W5;kfs::W6=nothing) where {U<:AbstractPhase,W6,W5<:Real,W1<:Real,W2<:Real,W3<:Real,W4<:Real, Q1<:AbstractArray,Q2<:ForwardDiff.Dual,Q3<:AbstractArray} #autodiff p
```

Comment:
Is there a better way to type these so they work properly with reverse mode differentiation (that might send something through that isn't an Array of ForwardDiff.Dual) as well as ForwardDiff?

Reply:
Thanks for the comment! I will dispatch getkfkrevs for both ReverseDiff and Tracker TrackedArray.